src/html-prep.ss

```diff
@@ -188,11 +193,13 @@
                                 (format "~a.html" root)
                                 (format "~a_~d.html" root n))))])
                 (open-output-file fn 'replace))])
-      (fprintf op "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\"\n  \"http://www.w3.org/TR/html4/loose.dtd\">")
+      (fprintf op "<!DOCTYPE html>~%")
       (fprintf op "<!-- DO NOT EDIT THIS FILE-->~%")
-      (fprintf op "<!-- Edit the .tex version instead-->~%~%")
+      (fprintf op "<!-- Edit the .stex version instead-->~%~%")
       (fprintf op "<html>~%")
       (fprintf op "<head>~%<title>~a</title>~%" title)
```

Comment:
Maybe we should add:
```
      (fprintf op "<meta charset=\"utf-8\">~%")
```
inside the header so that we don't get rendering like the following in some browsers:
![another-example](https://github.com/dybvig/stex/assets/18483559/aeaefefd-9501-4b97-8c57-19afaa412c7e)